Applications/Spreadsheet/Cell.h

```diff
@@ -117,6 +122,8 @@ struct Cell : public Weakable<Cell> {
     bool m_evaluated_externally { false };
     String m_data;
     JS::Value m_evaluated_data;
+    JS::Handle<JS::Cell> m_evaluated_data_handle;
+    JS::Handle<JS::Exception> m_js_exception;
```

Comment:
This does work but if we plan on holding on to a lot of these, it might be better to implement a `visit_edges` function that traces all the referenced JS cells instead of using handles (since handles have both a speed and space penalty in the GC.)

Reply:
Sure, but wouldn't the the speed savings be nullified by having to go through the entire hashmap of sheet cells and conditionally visiting stuff?

Reply:
Very possible. We can leave it using handles for now, no need to prematurely optimize :)
Perhaps something we'll come back to in the future

Reply:
Too late, I've already switched to `visit_edges()` and it looks much cleaner.